src/Microsoft.AspNetCore.SignalR/HubRouteBuilder.cs

```diff
@@ -9,20 +9,38 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// Maps incoming requests to <see cref="Hub"/> types.
+    /// </summary>
     public class HubRouteBuilder
     {
         private readonly ConnectionsRouteBuilder _routes;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HubRouteBuilder"/> class.
+        /// </summary>
+        /// <param name="routes">The routes builder.</param>
         public HubRouteBuilder(ConnectionsRouteBuilder routes)
         {
             _routes = routes;
         }
 
+        /// <summary>
+        /// Maps incoming requests with the specified path to the specified <see cref="Hub"/> type.
```

Comment:
I'm good with using "requests" here because the user is still working at an HTTP level in this type, since this is a route builder.